10.9-libcxx/stable/main/finkinfo/libs/qt5-qtwebengine.info

```diff
@@ -59,10 +59,18 @@ PatchScript: <<
 	perl -pi -e's|^linux:||g' tools/qmake/config.tests/srtp/srtp.pro
 	echo "INCLUDEPATH += %p/include" >> tools/qmake/config.tests/snappy/snappy.pro
 	echo "INCLUDEPATH += %p/include" >> tools/qmake/config.tests/srtp/srtp.pro
+	# Patch to use Fink python27 (on macOS 12+)
+	if [ ! -x /usr/bin/python ]; then
+		perl -pi -e 's|(python)( -c)|${1}2.7${2}|;s|(python)( ..[Qn][Tf][Wi])|${1}2.7${2}|' tools/qmake/mkspecs/features/functions.prf
+		perl -pi -e 's|(/usr/bin/env python)|${1}2.7|' tools/*scripts/*
+		perl -pi -e 's|(python)( ..QTW)|${1}2.7${2}|' src/core/gyp_run.pro
+		perl -pi -e 's|(/usr/bin/env python)|${1}2.7|' src/3rdparty/chromium/build/mac/*.py
+		perl -pi -e 's|(MacOSX.)(10)|${1}1[012]|' src/3rdparty/chromium/build/mac/find_sdk.py
+	fi
 <<
 NoSetCPPFLAGS: true
 NoSetCFLAGS: true
-NoSetCXXFLAGS: true
+SetCXXFLAGS: -std=gnu++99
```

Comment:
"`gnu++99`" isn't an actual standard; typo for "`gnu++98`"?

Reply:
```suggestion
SetCXXFLAGS: -std=gnu++98
```

Reply:
Probably confused with `c99/gnu99`

Reply:
> Probably confused with `c99/gnu99`

yeah that's what I was guessing